submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <!--header-->
+    <header class="container">
+      <!--header-menu-->
+      <nav class="menu">
+        <ul class="menu">
```

Comment:
Different by functionality elements with the same class name is a bad practice. Please fix it.